recipes/_configure_server.rb

```diff
@@ -166,9 +174,6 @@
       case key
       when 'general'
         server_stanzas['general']['guid'] = value['guid'] if value['guid']
```

Comment:
Let's also get rid of this... this was the case with Splunk 4, but ever since Splunk 5.0 and later guid doesn't exist in the server.conf, thus we don't need this delayed evaluation at all (with the addition of value procs :) )